Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAccountProtectionPolicyWindows10/MSFT_IntuneAccountProtectionPolicyWindows10.psm1

```diff
@@ -85,31 +85,42 @@ function Get-TargetResource
         Add-M365DSCTelemetryEvent -Data $data
         #endregion
 
-        $nullResult = $PSBoundParameters
-        $nullResult.Ensure = 'Absent'
-
-        $getValue = $null
-        #region resource generator code
-        $getValue = Get-MgBetaDeviceManagementConfigurationPolicy -DeviceManagementConfigurationPolicyId $Id -ErrorAction SilentlyContinue
-
-        if ($null -eq $getValue)
+        if ($PSBoundParameters.ContainsKey('Id'))
         {
-            Write-Verbose -Message "Could not find an Intune Account Protection Policy for Windows10 with Id {$Id}"
```

Comment:
Why were those messages removed in the resources? They were an indicator which policy with properties x and y was found.

Reply:
My point was more that the visibility of which resource is currently in progress is reduced. There is not really an indication of "I'm currently in the Intune Account Protection Policy for Windows10 and doing stuff there", it's a general message with "no real meaning".  I like the previous idea of the messages better, where we immediately saw where we are, if the policy was found (by id, or later by displayName) and so on.

Reply:
I will change the Verbose output back to about what it was!